### PR TITLE
Enable to load muted video

### PR DESCRIPTION
**Pull request by ronikar**
:)

### CODE REVIEW

File: src/layer/VideoOverlay.js [hunk 1]
```diff
@@ -71,6 +75,7 @@ export var VideoOverlay = ImageOverlay.extend({
 		if (!this.options.keepAspectRatio && vid.style.hasOwnProperty('objectFit')) { vid.style['objectFit'] = 'fill'; }
 		vid.autoplay = !!this.options.autoplay;
 		vid.loop = !!this.options.loop;
+		vid.muted = this.options.muted;
```
Comment (IvanSanchez) [inline src/layer/VideoOverlay.js:78]: Should be coerced into a `Boolean` with `!!`, as with the autoplay and loop options. Other than that, it's a good addition :-)

Comment (ronikar) [reply src/layer/VideoOverlay.js]: @IvanSanchez , I fixed it.

File: src/layer/VideoOverlay.js [hunk 2]
```diff
@@ -37,7 +37,11 @@ export var VideoOverlay = ImageOverlay.extend({
 		// @option keepAspectRatio: Boolean = true
 		// Whether the video will save aspect ratio after the projection.
 		// Relevant for supported browsers. Browser compatibility- https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit
```
Comment (Arimjj) [inline src/layer/VideoOverlay.js:39]: #7606

File: src/layer/VideoOverlay.js [hunk 3]
```diff
@@ -37,7 +37,11 @@ export var VideoOverlay = ImageOverlay.extend({
 		// @option keepAspectRatio: Boolean = true
 		// Whether the video will save aspect ratio after the projection.
 		// Relevant for supported browsers. Browser compatibility- https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit
-		keepAspectRatio: true
```
Comment (Arimjj) [inline src/layer/VideoOverlay.js:40]: True

Comment (Arimjj) [inline src/layer/VideoOverlay.js:40]: true

Comment (Arimjj) [inline src/layer/VideoOverlay.js:40]: True

Comment (Arimjj) [inline src/layer/VideoOverlay.js:40]: True

Comment (Arimjj) [inline src/layer/VideoOverlay.js:40]: Boolean = true

Comment (Arimjj) [inline src/layer/VideoOverlay.js:40]: Boolean = true

Comment (Arimjj) [reply src/layer/VideoOverlay.js:40]: @kshepard

**Comment by Queens666** [comment]
> :)

ronikar:muting-video-overlay

**Pull request merged**